perception_and_utils_root/perception_and_utils/utils/image_utils.py

```diff
@@ -158,3 +160,19 @@ def label_img(
         thickness,
         line_type,
     )
+
+
+def rotate_img(img: np.ndarray, num_of_rotation: int = 3) -> np.ndarray:
+    """
+    Rotate image in multiples of 90d degrees
+
+    Args:
+        img (np.ndarray): Image to be rotated
+        k (int, optional): Number of times to rotate by 90 degrees. Defaults to 3.
+
+    Returns:
+        np.ndarray: Rotated image
+    """
```

Comment:
Could you also add *why* we needed to add a specialized rotate_img rather than reusing numpy rot directly? I know there is a link in the body of the method, but it would help anyone looking at module definitions from VS Code/other language servers?

Reply:
Also add whether rotations are clockwise or anti-clockwise.

Reply:
Thanks for the suggestion, this makes sense!